airbyte-integrations/connectors/source-snowflake/source_snowflake/schema_builder.py

```diff
@@ -156,7 +156,7 @@ def format_field(field_value, field_type, local_time_zone_offset_hours=None):
             return field_value
 
         try:
-            if not isinstance(field_value, str) and len(field_value) == 0:
+            if not isinstance(field_value, str) or len(field_value) == 0:
```

Comment:
Can we update the comment `# maybe add warning` in the `except` statement? If I understand correctly, this is a date that would be empty, right? Why do we have this added condition? Is this a case we've seen? I would assume that since all our fields are nullable, we can return `None` in this case but if we don't expect this to happen, I would indeed add a log

Reply:
Corrected in the commit a84095b35c49a25d030b03862068c3c697774101